portfolio/src/main/webapp/pages/cat.html

```diff
@@ -6,12 +6,85 @@
     <link rel="stylesheet" href="../style.css">
     <script src="../script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>This is test 3</h1>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+  <body style="background-image: url('../images/paws_background.jpg');
```

Comment:
Please extract the styling out of the HTML content and into an appropriate stylesheet, per our style guide:
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns